index.html

```diff
@@ -91,27 +93,29 @@ <h2 class="work-title">Tonic</h2>
             </div>
 
             <!-- 2ND WORK SECTION -->
-            <div class="work-box">
+            <div class="work-box reverse">
```

Comment:
- [ ] It would be good if you CSS `:nth-child(even)` Selector instead of manually adding css classes. Let's say, you want to add a project a month from now. This way, you have to come to this code and assign the appropriate css classes. please refer [this](https://www.w3schools.com/cssref/sel_nth-child.asp) link.

Reply:
thanks changes resolved